packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -127,7 +127,15 @@ export default function renderTypeCell(event: Event, clusterId: string) {
         <StyledEventType>
           <a
             title="Open Session Player"
-            href={cfg.getPlayerRoute({ clusterId, sid: event.raw.sid })}
+            href={cfg.getPlayerRoute(
+              {
+                clusterId,
+                sid: event.raw.sid,
+              },
+              {
+                recordingType: 'ssh',
+              }
```

Comment:
seems awkward, is there a reason why we can't combine `recordingType` with the first param object?

Reply:
https://github.com/gravitational/webapps/pull/572#discussion_r797725636